partcad/src/partcad/render.py

```diff
@@ -19,6 +19,6 @@
     "strokeColor": [64, 192, 64],
     "hiddenColor": [32, 64, 32],
     "showHidden": False,
-    # "marginLeft": 0,
-    # "marginTop": 0,
+    "marginLeft": 0,
```

Comment:
None means central alignment. 0 means left alignment. They are not the same. Please, default to None. Run "render_all_examples.sh" in the examples folder to ensure that the other images were not impacted when they should not be.